public/electron.js

```diff
@@ -0,0 +1,53 @@
+// From: https://www.section.io/engineering-education/desktop-application-with-react/
+const path = require('path')
+
+const { app, BrowserWindow } = require('electron')
+const isDev = require('electron-is-dev')
+
+const { autoUpdater } = require('electron-updater')
+
+function createWindow() {
+  // Create the browser window.
+  const win = new BrowserWindow({
+    width: 1024,
+    height: 768,
+    webPreferences: {
+      nodeIntegration: true,
+    },
+    icon: path.join(__dirname, './app-icons/Icon-512x512.png'),
+  })
+
+  // and load the index.html of the app.
+  // win.loadFile("index.html");
+  win.loadURL(
+    isDev
+      ? 'http://localhost:3000'
+      : `file://${path.join(__dirname, '../build/index.html')}`
```

Comment:
very minor comment but i find things like this much easier to read if stored into a variable instead of done inside the function call. 

```suggestion
  const hostUrl = isDev ? 'http://localhost:3000' : `file://${path.join(__dirname, '../build/index.html')}`
  
  win.loadURL(hostUrl)
```